frontend/src/store/account.js

```diff
@@ -189,13 +189,14 @@ const actions = {
                 teamId = null
                 teamSlug = teamIdMatch[1]
             }
-
             try {
-                const team = await teamApi.getTeam(teamId || { slug: teamSlug })
-                const teamMembership = await teamApi.getTeamUserMembership(team.id)
-                state.commit('setTeam', team)
-                state.commit('setTeamMembership', teamMembership)
-
+                // Some routes will call `setTeam` once they figure out who they belong to.
+                if (!/^\/(application|device|instance)\//.test(router.currentRoute.value.path)) {
```

Comment:
I think this needs to be more explicit about what this doing and why. Right now (without the context of this PR) it reads not applications device instance in current path, which is kind of unclear as to what the objective is...

Reply:
I've added a longer comment to make it clearer. 
https://github.com/FlowFuse/flowfuse/pull/3235/commits/d524e15f983f600f70c1e27d8c62eed5a6344dcf